runtime/lua/vim/lsp/util.lua

```diff
@@ -24,6 +24,7 @@ local border_size = {
   rounded = { 2, 2 },
   solid = { 2, 2 },
   shadow = { 1, 1 },
+  bold = { 3, 3 },
```

Comment:
How can border size be 3? The are only two borders in each direction.
```suggestion
  bold = { 2, 2 },
```